public/manifest.json

```diff
@@ -1,25 +1,25 @@
 {
-  "short_name": "React App",
-  "name": "Create React App Sample",
+  "short_name": "What's Near Me",
+  "name": "What's Near",
   "icons": [
     {
-      "src": "favicon.ico",
+      "src": "map-marked-alt-solid.svg",
       "sizes": "64x64 32x32 24x24 16x16",
       "type": "image/x-icon"
     },
     {
-      "src": "logo192.png",
+      "src": "map-marked-alt-solid.png",
       "type": "image/png",
       "sizes": "192x192"
     },
     {
-      "src": "logo512.png",
+      "src": "map-marked-alt-solid.png",
       "type": "image/png",
       "sizes": "512x512"
     }
   ],
   "start_url": ".",
   "display": "standalone",
-  "theme_color": "#000000",
+  "theme_color": "#5B1B52",
   "background_color": "#ffffff"
```

Comment:
I've never used `public/manifest.json` 😮😮😮. But, thank you for sharing some resources! I learned a lot 😄!!!